### DISCUSSION TITLE
Any timeframe for when the OS kernel would be updated to 6.4?

### DISCUSSION THREAD

**Discussion by tessharp** [home-assistant/operating-system#2897] [Q&A] [answered]
I am using a WiFi adapter that is not Plug and Play on the 6.1.# Kernel, but it is on 6.4 - What is the roadmap for these upgrades?

PS: I am using a WiFi adapter because I am keeping IoT completely separate from my home network, and while my router supports OpenWrt, it only has a hub, rather than a switch for port management, cannot assign a dedicated VLAN for a single port.

**Comment by sairon** [comment, marked as answer]
HAOS sticks to LTS kernel and Buildroot versions and there are no plans from departing that approach, so at least until new kernel version is promoted to LTS, it's unlikely the kernel will be updated to any newer minor release. Since 6.1 was released last December and promoted to LTS in February, I expect the next LTS won't be available within the following few months at least.

What WiFi adapter are you using? In some occasions when the complexity is not so high and there's no risk of breakage, we could consider backporting the device support to our patchset. For that it'd be best if you created an issue in the repo with details what card you're seeking support for. Thanks!

**Reply by tessharp** [reply]
I'm using a Netgear A8000. Will create an issue then!

**Reply by marcuscps** [reply]
Kernel 6.6 has been promoted to LTS on 2023-10-29. Any plans to update to it?

BTW, I highly appreciate the work you are doing! :) 

-- Edit:
Now I understand the LTS version is defined by Buildroot, not the Kernel Releases.
Apologies for my confusion.

**Reply by agners** [reply]
We'll probably schedule it with the major update to HAOS 12.0, which brings U-Boot and Buildroot update as well (February timeframe).

That said, since there are reasons, we might consider update the kernel earlier this time around.
